draft-ietf-quic-recovery.md

```diff
@@ -217,7 +217,7 @@ not available.
 
 ### Clearer Loss Epoch
 
-QUIC ends a loss epoch when a packet sent after loss is declared is
+A loss epoch ends when a packet sent after loss is first declared is
```

Comment:
I don't understand this sentence. There are too many "is".

Reply:
Fair, I tried to rephrase this sentence another time.  PTAL